sepal_ui/mapping/fullscreen_control.py

```diff
@@ -97,7 +78,12 @@ def toggle_fullscreen(self, *args) -> None:
         # change button icon
         self.w_btn.children[0].children = [self.ICONS[self.zoomed]]
 
-        # zoom
-        self.template.send({"method": self.METHODS[self.zoomed], "args": []})
+        (
+            self.m.add_class("full-screen-map")
+            if self.zoomed
+            else self.m.remove_class("full-screen-map")
+        )
```

Comment:
```suggestion
        if self.zoomed:
            self.m.add_class("full-screen-map")
        else:
            self.m.remove_class("full-screen-map")
```

I usually don't like it but it's currently shorter. I will request some extra method in ipywidget: https://github.com/jupyter-widgets/ipywidgets/issues/3905